src/stac-wasm/stac_wasm.js

```diff
@@ -0,0 +1,5 @@
+import * as wasm from "./stac_wasm_bg.wasm";
+export * from "./stac_wasm_bg.js";
+import { __wbg_set_wasm } from "./stac_wasm_bg.js";
+__wbg_set_wasm(wasm);
+wasm.__wbindgen_start();
```

Comment:
Is this auto-generated? You should never have to look into `_bg.js` yourself

Reply:
Yeah it's from wasm-pack